work.html

```diff
@@ -225,42 +225,53 @@ <h3 class="header-3">Skills</h3>
         </div>
       </div>
     </article>
-    <article class="contact">
-      <div class="grid-container">
-        <div class="grid-item grid-item-1 flex-container contact_flex">
-          <h2 class="contact_header flex-item-1">Contact Me</h2>
-          <p class="contact_paragraph flex-item-2">
-            If you have an application you are interested in developing, a
-            feature that you need built or a project that needs coding. I’d love
-            to help with it
-          </p>
-          <form class="contact_form flex-item-3" action="#">
-            <input
-              class="contact_input"
-              type="text"
-              name="name"
-              id="name"
-              placeholder="Yeremias"
-            />
-            <input
-              class="contact_input"
-              type="email"
-              name="email"
-              id="email"
-              placeholder="natajayanj14@gmail.com"
-            />
-            <textarea
-              class="contact_textarea"
-              name="textarea"
-              id="textarea"
-              cols="30"
-              rows="8"
-              placeholder="Write your message here"
-            ></textarea>
-            <input class="contact_submit" type="button" value="Submit" />
-          </form>
+    <footer>
+      <article class="contact">
+        <div class="grid-container">
+          <div class="grid-item grid-item-1 flex-container contact_flex">
+            <h2 class="contact_header flex-item-1">Contact Me</h2>
+            <p class="contact_paragraph flex-item-2">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+            <form
+              class="contact_form flex-item-3"
+              action="https://formspree.io/f/mwkywvdn"
+              method="post"
+            >
+              <input
+                class="contact_input"
+                type="text"
+                name="name"
+                id="name"
+                placeholder="write your name"
+                maxlength="30"
+                required
+              />
+              <input
+                class="contact_input"
+                type="email"
+                name="email"
+                id="email"
+                placeholder="write your email"
+                required
+              />
+              <textarea
+                class="contact_textarea"
+                name="textarea"
+                id="textarea"
+                cols="30"
+                rows="8"
+                placeholder="Write your message here"
+                maxlength="500"
+                required
+              ></textarea>
+              <input class="contact_submit" type="submit" value="Submit" />
+            </form>
```

Comment:
- [ ] **[REQUIRED CHANGES]**

> || **YOUR DESIGN**|| **FIGMA DESIGN** ||
> |-|---------|-|---------|-|
> || ![space](https://user-images.githubusercontent.com/59397159/165367610-a65e6678-de84-4e7e-830a-beeeac1da662.png) || ![space2](https://user-images.githubusercontent.com/59397159/165367744-1975f8c3-cbe6-415c-be25-dda1705570b3.png) || 

- Please, kindly remove the excess space, add the line and change the text content of your button to match that of the Figma design. 

- Please update your readme by doing the following:

1. change your project title
2. Remove the `author header` after the `Built with` header and `Project Name` header.
3. Add the second author that contributed to this project.
4. Remove the `Deployment`, `Usage`, `Run test`, `Install`, and `setup` as they are not descriptive enough.
5. Update your `Prerequisite`  e.g 
```

## Getting Started

To get a local copy up and running follow these simple example steps.

### Getting started
 - A text editor(preferably Visual Studio Code)

### Install
  -  [Git](https://git-scm.com/downloads)
### Usage
#### Clone this repository

```bash
$  https://github.com/waqaskanju/Portfolio.git
$ cd Portfolio
```
```

Reply:
Thanks for your suggestions. Form and read me file updated as per your instructions.